projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/ui/ConnectionPanel.kt

```diff
@@ -71,22 +71,21 @@ class ConnectionPanel(private val resolver: AsyncHostResolver) : JPanel(),
 
   private fun getToggleButtonIcon() = if (isProjectorStopped()) getMenuOpenIcon() else AllIcons.Actions.Exit
 
-  private fun getMenuOpenIcon(): Icon {
+  private fun getMenuOpenIcon(): Icon = try { // <= 202  compatibility
```

Comment:
This comment isn't such clear as comments in `getVMOptionsWriteFile`: there every branch is annotated with corresponding platform versions. I suggest doing it like that in all such places.